tests/Unit/Domain/CoordinateMaps/Test_CylindricalSide.cpp

```diff
@@ -0,0 +1,174 @@
+// Distributed under the MIT License.
```

Comment:
[nitpick] I think this commit should probably be squashed into the previous one, under the philosophy that the feature and test should be present in the same commit.